IDS/Config/Config.ini.php

```diff
@@ -19,8 +19,8 @@
     
     ; in case you want to use a different HTMLPurifier source, specify it here
     ; By default, those files are used that are being shipped with PHPIDS
-    HTML_Purifier_Path	= vendors/htmlpurifier/HTMLPurifier.auto.php
-    HTML_Purifier_Cache = vendors/htmlpurifier/HTMLPurifier/DefinitionCache/Serializer
+    HTML_Purifier_Path	= [civicrm.root]/vendor/ezyang/htmlpurifier/library/HTMLPurifier.auto.php
+    HTML_Purifier_Cache = [civicrm.root]/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer

```

Comment:
@totten @mfb note that because i am unsure of the directory structure for within D8 i have aimed to go with a relative path here and patched up the IDS/Monitor.php to support the relative path